addons/web/static/src/core/hotkeys/hotkey_service.js

```diff
@@ -313,8 +310,9 @@ export const hotkeyService = {
                     unregisterHotkey(token);
                 };
             },
+            overlayModifier: hotkeyService.overlayModifier,
```

Comment:
Is this only useful to display right hotkey in the command palette ?
If so, this is really a pity :-(

Couldn't we:
- either import the hotkeyService description (or get it from the registry) to access the overlayModifier directly in the command palette stuffs
- either expose something like getKeysToPress(hotkey, withOverlayModifier = true) ?

I think I prefer option A of what I suggest.

Reply:
remove this